ADO/ADOLibrary/Interfaces/IOrderRepository.cs

```diff
@@ -0,0 +1,15 @@
+﻿using ADOLibrary.Models;
+using System.Data;
+
+namespace ADOLibrary.Interfaces
+{
+    public interface IOrderRepository
+    {
+        public Task Insert(Order product);
```

Comment:
You may consider to make generic as most of method are the same for both IOrderRepository and IProductRepository. I assume it was not created by you. So, just keep it this suggestion in your mind.